Makefile

```diff
@@ -30,3 +30,9 @@ bash/fmt:
 
 bash/fmt/check:
 	shfmt -d $(PWD)/rootfs
+
+apk-update:
+	@echo package_repos.md5 old $$(cat package_repos.md5 || echo '<not found>')
+	@docker run --rm -e CLUSTER=galaxy $(DOCKER_IMAGE_NAME) -c \
+	'apk update >/dev/null && md5sum /var/cache/apk/APKINDEX.* | md5sum | colrm 33' > package_repos.md5
```

Comment:
```suggestion
	'apk update >/dev/null && md5sum /var/cache/apk/APKINDEX.* | md5sum | colrm 33' > apkindex.md5
```

Reply:
This really has nothing to do with `packages.txt` and more to do with the `APKINDEX`. Something like `APKINDEX.md5` or `.APKINDEX` or `.APKINDEX.md5` would be better names IMO.

Reply:
Using `APKINDEX.md5` has a high probability of interfering with, or at least being mistaken for, something used by `apk`. Since the `apk` documentation is sparse, I hate to do anything potentially confusing like that.

I'm not particularly advocating for `package_repos.md5`, but I do want something that will not be mistaken for part of the official `apk` integrity checks.